web_action_conditionable/__manifest__.py

```diff
@@ -1,13 +1,12 @@
-# -*- coding: utf-8 -*-
-
 {
     "name": 'web_action_conditionable',
-    "version": "8.0.0.1.0",
+    "version": "10.0.1.0.0",
     "depends": [
         'base',
         'web',
     ],
     'data': ['views/view.xml'],
-    "author": "Cristian Salamea,Odoo Community Association (OCA)",
-    'installable': False,
+    "author": "ACSONE SA/NV,Cristian Salamea,Odoo Community Association (OCA)",
```

Comment:
I usually consider significant contributions warrant adding a company/person as author. 

This PR looks like a simple migration, so it may not be necessary to add ACSONE here.